src/locales/en/translation.json

```diff
@@ -60,7 +60,8 @@
       "merchants": "WAYS TO DONATE",
       "waystodonate": {
         "donate": "DONATE",
-        "gift-a-meal": "GIFT-A-MEAL"
+        "gift-a-meal": "GIFT-A-MEAL",
+        "cookbook": "COOKBOOK"
```

Comment:
https://media.discordapp.net/attachments/817849576672985098/835166029146488892/unknown.png
Should be SCL COOKBOOK instead of COOKBOOK

Reply:
changed